plugins/open-feedback/src/components/OpenFeedbackPage/OpenFeedbackPage.tsx

```diff
@@ -3,18 +3,23 @@ import { Typography, Grid } from '@material-ui/core';
 import { InfoCard, Header, Page, Content } from '@backstage/core-components';
 import { FeedbackCards } from '../OpenFeedbackCard/OpenFeedbackCard';
 
-export const OpenFeedbackPage = () => (
+export interface OpenFeedbackPageProps {
+  title?: string;
+  subtitle?: string;
+}
+
+export const OpenFeedbackPage = ({
+  title = 'Welcome to OpenFeedback!',
+  subtitle = 'Collected Feedback for your Backstage App!',
```

Comment:
Would it be okay to add a small snippet in the README on how you'd update the title and subtitle to something else if you want in the `App.tsx` like:

```ts
    <Route path="/open-feedback" element={<OpenFeedbackPage title='My Super Feedback Title!' subtitle='A Super Subtitle!'/>} />
 ```

WDYT? 😅